pretrain/tinyllama.py

```diff
@@ -17,6 +17,7 @@
 from lightning.fabric.utilities.throughput import ThroughputMonitor, measure_flops
 from lightning.pytorch.loggers import WandbLogger
 from torch.utils.data import DataLoader
+from torchmetrics.aggregation import RunningMean
```

Comment:
Note: Instead of using torchmetrics as a dependency, we could also implement a simple RunningMean class here in lit-gpt. Let me know what you prefer.

Reply:
Lit-gpt installs lightning which includes pytorch lightning which includes torchmetrics.

So you are not adding any weight here. You are fine.

Reply:
Thank you Carlos!